ndmg/scripts/ndmg_cloud.py

```diff
@@ -56,7 +55,7 @@ def batch_submit(
     log=False,
     bg=False,
     modif="",
-    reg_style=""
+    reg_style="",
```

Comment:
Why is there a comma here now?

Reply:
I don't think I added that myself, so it might be a Black thing. Entirely stylistic

Reply:
Please confirm that it doesn't break anything